images/dotnet/.dockerignore

```diff
@@ -0,0 +1,20 @@
+# Ignore .NET Core build output directories
+bin/
```

Comment:
needs the mac os files excluded (see how .dsstore and others are excluded in other dockerignore file examples)